meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-mkimage/imx-boot_%.bbappend

```diff
@@ -58,7 +62,7 @@ do_deploy_append() {
     for target in ${IMXBOOT_TARGETS}; do
         if [ "${target}" = "flash_evk_spl" ]; then
             # Also create imx-boot link with the machine name
-            ln -sf ${BOOT_CONFIG_MACHINE}-${target} ${DEPLOYDIR}/${BOOT_NAME}-${MACHINE}
+            #ln -sf ${BOOT_CONFIG_MACHINE}-${target} ${DEPLOYDIR}/${BOOT_NAME}-${MACHINE}
```

Comment:
For Apalis iMX8 this symlink is created somewhere else, and this causes build failure, so I've just commented that for now